setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""Installation and deployment script."""
+
+try:
+  from setuptools import find_packages, setup
+except ImportError:
+  from distutils.core import find_packages, setup
+
+from docker_explorer import __version__ as de_version
+
+
+de_description = 'Docker forensic analysis tool.'
+
+de_long_description = (
+    'docker-explorer is a tool to help forensic analysis of offline Docker '
+    'installations')
+
+setup(
+    name='docker-explorer',
```

Comment:
doesn't the `-` cause issues?

https://www.python.org/dev/peps/pep-0008/#package-and-module-names

Reply:
fixed